lib/google/gax/api_callable.rb

```diff
@@ -112,14 +135,17 @@ def initialize(a_func, request_page_token_field,
       end
 
       # Initiate the streaming with the requests and keywords.
+      # @param page_token [Object]
+      #   The initial page_token or nil.
```

Comment:
I think "initial page_token" could be misinterpreted to mean "the page token for the initial page". (E.g., Python has a constant `INITIAL_PAGE`, which means the page token for the initial page overall.)

How about, "The page_token for the for the first page to be streamed, or nil"?